src/components/implementation/no_interface/cFE_booter/sl_mod_fprr.c

```diff
@@ -0,0 +1,120 @@
+#include <llprint.h>
```

Comment:
I forget why this is here and what the difference is between this and `components/lib/sl/sl_mod_fprr.c`.

Reply:
If it's identical, perhaps soft-link.

Reply:
It's not identical, we need more priorities.